Gemfile.lock

```diff
@@ -110,8 +99,6 @@ GEM
     faker (2.14.0)
       i18n (>= 1.6, < 2)
     ffi (1.13.1)
-    font-awesome-rails (4.7.0.5)
```

Comment:
Wait so did you delete these gems?

Reply:
No. I didnt touch the gem file

Reply:
hahah okay. I was just asking just to make sure, because I didn't know if Zach or someone had told you to while doing the carousel :)

Reply:
lol. nope